go.mod

```diff
@@ -45,4 +60,6 @@ replace (
 	github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.29.0
 )
 
+replace github.com/codeready-toolchain/toolchain-common => github.com/xcoulon/toolchain-common v0.0.0-20200131154201-fea8485ab790
```

Comment:
You forgot to delete this replace.

Reply:
no, I pushed without committing the removal, first! 🤦‍♂

Reply:
done in https://github.com/codeready-toolchain/member-operator/pull/121/commits/8398463cd4b37ca57718e879cf8110ce059cd84c